mailet/standard/src/main/java/org/apache/james/transport/mailets/StripAttachment.java

```diff
@@ -187,32 +184,37 @@ public void init() throws MailetException {
     }
 
     /**
-     * Service the mail: scan it for attchemnts matching the pattern, store the
-     * content of a matchin attachment in the given directory.
+     * Service the mail: scan it for attachments matching the pattern, store the
+     * content of a matching attachment in the given directory.
      * 
      * @param mail
      *            The mail to service
      * @throws MailetException
      *             Thrown when an error situation is encountered.
      */
+    @Override
     public void service(Mail mail) throws MailetException {
-        MimeMessage message;
+        // All MIME messages with an attachment are multipart, so we do nothing
```

Comment:
In my opinion this comment explains something that is very simple (Attachments implies multipart).

The condition after it also well express the fact we keep only MimeMultiPart as we are not interrested in simple messages.